src/cmds/scripts/win_postinstall.py

```diff
@@ -368,6 +420,11 @@ def main():
             username = arg
         elif opt in ('-p', '--passwd'):
             password = arg
+        elif opt in ('-t', '--type'):
+            installtype = arg
```

Comment:
Just suggestion (take it or leave it up to you 😄 ): Assign installtype after validating arg (aka after below if block)

Reply:
I am taking this suggestion, this will reduce execution time and space in the negative case.